### PR TITLE
resolved merge conflicts with ace pull req

### PR DESCRIPTION
**Pull request by kyle-richardson**
login and create account forms mostly complete.  need to add functionality with back end

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,14 +3,23 @@
...
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.25",
+    "@fortawesome/free-solid-svg-icons": "^5.11.2",
+    "@fortawesome/react-fontawesome": "^0.1.6",
     "@material-ui/core": "^4.5.1",
     "axios": "^0.19.0",
+    "extends": "^0.0.4",
     "formik": "^1.5.8",
+    "less-watch-compiler": "^1.14.1",
```
Comment (AceIsHuman) [inline package.json]: `less-watch-compiler` should be installed globally on your computer.

`yarn global add less-watch-compiler` or
`npm install -g less-watch-compiler`

File: package.json [hunk 2]
```diff
@@ -3,14 +3,23 @@
...
+    "material-ui": "^0.20.2",
```
Comment (AceIsHuman) [inline package.json]: We can delete this line. We are using `@material-ui/core`

 Side note, Material UI also comes with Icons you can use. 
https://material-ui.com/components/material-icons/

Comment (kyle-richardson) [reply package.json]: great, just played with material UI icons.  works just as well, so I removed the font awesome libraries from package.json locally.

File: package.json (hunk 1 above)
Comment (kyle-richardson) [reply package.json]: right you are.  not sure why I added it with just yarn add.  nice catch, need to remove as much bloat as we can from libraries

**Pull request merged**